microsite/static/css/custom.css

```diff
@@ -1210,7 +1210,7 @@ code {
 }
 
 .nav-footer .copyright {
-  width: 600px;
```

Comment:
Hmm - can we do `width:100%` and then padding left and right 8px or something for gutters?

Reply:
@benjdlambert, yeah! I'll update it